### PR TITLE
Open guided install nerdlet in a card

### PR DESCRIPTION
**Pull request by mbazhlekova**
## Description

The guided install will be opened in a card on top of the explorer nerdlet.

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

File: src/utils/get-pack-nr1-url.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 const NERDLET_PATH = `launcher/nr1-core.explorer/`;
 
 /**
- * @param {string} packId The ID for an observability pack.
+ * @param {string} quickstartId The ID for an observability pack.
  * @param {boolean} [debug] If set to true, this will add `packages=local`.
  * @returns {string} The URL for the pack details within the platform.
  */
-const getPackNr1Url = (quickstartId, nerdletId, debug = false) => {
+export const getPackNr1Url = (quickstartId, debug = false) => {
```
Comment (aswanson-nr) [inline src/utils/get-pack-nr1-url.js:17]: This file had to be adjusted to run on Node.js as well, could you change this to a module export?
```suggestion
const getPackNr1Url = (quickstartId, debug = false) => {
```

File: src/utils/get-pack-nr1-url.js [hunk 2]
```diff
@@ -24,9 +30,39 @@ const getPackNr1Url = (quickstartId, nerdletId, debug = false) => {
...
-  return url.href;
+  return packDetailsUrl.href;
 };
 
-module.exports = getPackNr1Url;
+/**
+ * @param {boolean} [debug] If set to true, this will add `packages=local`.
+ * @returns {string} The URL for the pack details within the platform.
+ */
+export const getGuidedInstallStackedNr1Url = (debug = false) => {
```
Comment (aswanson-nr) [inline src/utils/get-pack-nr1-url.js:45]: Here as well
```suggestion
const getGuidedInstallStackedNr1Url = (debug = false) => {
```

File: src/utils/get-pack-nr1-url.js [hunk 3]
```diff
@@ -24,9 +30,39 @@ const getPackNr1Url = (quickstartId, nerdletId, debug = false) => {
...
+  const packDetailsUrl = new URL(
+    `${NERDLET_PATH}?${local}pane=${hash}`,
+    NR1_URL
+  );
 
-  return url.href;
+  return packDetailsUrl.href;
 };
 
-module.exports = getPackNr1Url;
```
Comment (aswanson-nr) [inline src/utils/get-pack-nr1-url.js:32]: This is the export statement we'd like to keep. It'll just need to be changed to a named export

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
The changes look good! Just had a few comments about sticking with syntax that works on Node.js

**Review by aswanson-nr — DISMISSED** [review]

**Comment by mbazhlekova** [comment]
@aswanson-nr I haven't worked with modules in a while and it took me longer than I expected to get it working but it's good now.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**